timelock-corruption-detection/src/main/conjure/timelock-history.yml

```diff
@@ -0,0 +1,52 @@
+types:
+  imports:
+    PaxosValue:
+      base-type: any
+      external:
+        java: com.palantir.paxos.PaxosValue
+    PaxosAcceptorState:
+      base-type: any
+      external:
+        java: com.palantir.paxos.PaxosAcceptorState
+    NamespaceAndUseCase:
+      base-type: any
+      external:
+        java: com.palantir.paxos.NamespaceAndUseCase
+    Long:
+      base-type: any
+      external:
+        java: java.lang.Long
+  definitions:
+    default-package: com.palantir.timelock.history
+    objects:
+      HistoryQuery:
+        fields:
+          namespaceAndUseCase: NamespaceAndUseCase
+          seq: Long
+      PaxosLogWithAcceptedAndLearnedValues:
+        fields:
+          acceptedState: optional<PaxosAcceptorState>
+          paxosValue: optional<PaxosValue>
+          seq: Long
+      LogsForNamespaceAndUseCase:
+        fields:
+          namespaceAndUseCase: NamespaceAndUseCase
+          logs: list<PaxosLogWithAcceptedAndLearnedValues>
+      PaxosHistoryOnRemote:
+        fields:
+          logs: list<LogsForNamespaceAndUseCase>
+
+services:
+  TimeLockPaxosHistoryProvider:
+    name: TimeLock Paxos History Provider Service
+    default-auth: header
+    package: com.palantir.timelock.history
```

Comment:
We should standardise on the package to be used (I think, unless I'm missing something - you've used both `com.palantir.timelock.history` and `com.palantir.history`). I think the former is the correct one (i.e. no changes here, but we might want to move the others around).